src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/ProviderBase/DbMetaDataFactory.cs

```diff
@@ -549,7 +550,7 @@ private void LoadDataSetFromXml(Stream XmlStream)
         {
             _metaDataCollectionsDataSet = new DataSet();
             _metaDataCollectionsDataSet.Locale = System.Globalization.CultureInfo.InvariantCulture;
-            _metaDataCollectionsDataSet.ReadXml(XmlStream);
+            _metaDataCollectionsDataSet.ReadXml(XmlReader.Create(XmlStream, new XmlReaderSettings() { XmlResolver = null }));
```

Comment:
Roslyn warning: `warning CA3075: Unsafe overload of 'ReadXml' method`
https://docs.microsoft.com/en-us/visualstudio/code-quality/ca3075?view=vs-2019

Both ReadXml overloads with a Stream argument produce this warning.